app/models/requests.js

```diff
@@ -18,13 +18,13 @@ const requestSchema = Schema({
     },
     status: {
         type: String,
-        "default": "pending",
+        default: "pending",
```

Comment:
@ruanres e @saviofelixmuniz, quero as suas opiniões sobre o melhor approach aqui.
Dado que agora será um administrador que definirá a prioridade da requisição, a requisição ficará sem essa informação até o momento que for avaliada.
Vocês acham melhor deixar dessa forma ou eliminar esse enum undefined e deixar o default como undefined?